tests/test_utils.py

```diff
@@ -303,3 +306,437 @@ def check_multistream_classic(fnames):
+
+            # read in data with -9999.0000 as NaNs
+            ecflux_df = pd.read_csv(os.path.join(obs_dir, subdir, filename),
+                    skiprows=0,
+                    delim_whitespace=True,
+                    header=None,
+                    names=names,
+                    na_values=-9999.0000)
+
+            ecflux_df['DATES'] = pd.to_datetime(ecflux_df.Year * 10000 +
+                                         ecflux_df.Month * 100 +
+                                         ecflux_df.Day,format='%Y%m%d')
+
+        else:
+            subdir_files = False
+
+        # load VIC 4.2 simulations
+        ebal_42_file = 'en_bal_%s_%s' % (lat, lng)
+
+        try:
+
+            ebal_42 = pd.read_csv(os.path.join(vic_42_dir,
+                                            ebal_42_file),
+                                            sep='\t',
+                                            skiprows=5)
+
+            # remove comment sign from column names in DataFrame
+            ebal_42 = ebal_42.rename(columns=lambda x: x.replace('#', ''))
+            ebal_42 = ebal_42.rename(columns=lambda x: x.replace(' ', ''))
+
+            ebal_42['DATES'] = pd.to_datetime(ebal_42.YEAR * 10000 +
+                                     ebal_42.MONTH * 100 +
+                                     ebal_42.DAY, format='%Y%m%d')
+        except OSError:
+            lat_issue=True
+
+        # load VIC 5.0 simulations
+        ebal_50_file = 'en_bal_%s_%s.txt' %(lat, lng)
+
+        try:
+
+            # load VIC 5 simulations
+            ebal_50 = pd.read_csv(os.path.join(vic_50_dir,
+                                 ebal_50_file),
+                                 skiprows=3,
+                                 sep='\t')
+
+            # remove space from column names in DataFrame
+            ebal_50 = ebal_50.rename(columns=lambda x: x.replace(' ', ''))
+
+            ebal_50['DATES'] = pd.to_datetime(ebal_50.YEAR * 10000 +
+                                         ebal_50.MONTH * 100 +
+                                         ebal_50.DAY, format='%Y%m%d')
+
+        except OSError:
+            print("this site has a lat/lng precision issue")
+            lat_issue=True
+            # To-Do: deal with lat/lng precision issue
+
+        # load VIC 5.0.x simulations
+        ebal_50x_file = 'en_bal_%s_%s.txt' %(lat, lng)
+
+        try:
+
+            # load VIC 5.x simulations
+            ebal_50x = pd.read_csv(os.path.join(result_dir,
+                                 ebal_50x_file),
+                                 skiprows=3,
+                                 sep='\t')
+
+            # remove space from column names in DataFrame
+            ebal_50x = ebal_50x.rename(columns=lambda x: x.replace(' ', ''))
+
+            ebal_50x['DATES'] = pd.to_datetime(ebal_50x.YEAR * 10000 +
+                                             ebal_50x.MONTH * 100 +
+                                             ebal_50x.DAY, format='%Y%m%d')
```

Comment:
factor out all the data munging into separate functions by dataset type.  This would also be a good place to use `read_vic_ascii()` in tonic.